ansible.cfg

```diff
@@ -1,7 +1,4 @@
 [defaults]
-# Set the logging verbosity level
-verbosity = 2
-
```

Comment:
Setting this causes `ansible-galaxy` to generate invalid JSON output with `--format=json`. I consider this a [bug in ansible-galaxy](https://github.com/ansible/galaxy/issues/3490).